docs/modules/documentation/containers/alert/alert-docs.component.html

```diff
@@ -1,56 +1,72 @@
+    { name: 'mousePersist', description: 'Boolean. When set to true, the alert will remain visible as long as the mouse is hovered over it. Default is false.'},
+    { name: 'duration', description: 'Number. Dictates how long in milliseconds the alert will be visible. Default is 10000.'},
+    { name: 'ariaLabel', description: 'String. Aria-label attribute for the alert component.' },
+    { name: 'ariaLabelledBy', description: 'String. Aria-labelledby attribute for the alert component.' },
+    { name: 'width', description: 'String. Width of the alert. Supports all css units.' },
+    { name: 'message', description: 'String. Alternative way of displaying the alert message.' }
   ],
   outputs: [
-    { name: 'close', description: 'Event - Emitted when the close button is clicked.' }
+    { name: 'close', description: 'Emitted when the close button is clicked.' }
   ]
 }"></properties>
 
 <separator></separator>
 
 <description>
-    Alerts can be opened one of two ways, either by passing a template reference variable or by passing a component reference as content.  For simple cases where only one alert will be shown
-    at a time, we recommend the former (see example 1).  However, when multiple alerts will be shown simultaneously, we recommend the latter (example 2), as a new instance of the alert
-    component will be created each time the <code>alertService.open</code> function is called.  Using the latter method allows reuse of the same alert component with different parameters
-    and will result in better alert stacking behavior.
+    Alerts can either be inline or appended to the body as part of an overlay. The first behaviour is easy to achieve and only requires
+    making use of the <code>&lt;fd-alert&gt;</code> component. However, many input properties such as <code>duration</code> and
+    <code>persist</code> are not applied by default. To enable them, call the <code>open()</code> method of the alert component. This extra step
+    was added to facilitate the use of basic inline alerts.
+
+    <br /><br />
+    For overlay alerts, use of the <code>AlertService</code> is required. These can be loaded by providing the service with a component type,
+    a template, or a string. Various means of accessing and interacting with the generated material is provided and showcased in examples below.
 </description>
 
 <separator></separator>
 
-<h2>Alert opened by template reference variable</h2>
+<h2>Alert Types</h2>
+<description>A number of alert types are available. They offer preset styling for different contextual situations.</description>
 <component-example [name]="'ex1'">
   <fd-alert-example></fd-alert-example>
 </component-example>
 <code-example [code]="alertHtml" [language]="'HTML'"></code-example>
-<code-example [code]="alertTs" [language]="'TypeScript'"></code-example>
 
 <separator></separator>
 
-<h3>Alert opened by component reference as content</h3>
+<h2>Alert as Overlay</h2>
```

Comment:
What do you mean as Overlay?
It would be more clear for me if it was something like `Opening Alerts from Services` or `Controlling alerts Programmatically` or `The Alert Service` or... I dunno something like this.

Reply:
I'll think about a better way to phrase it, wasn't really sure how to put it